common/txmgr/types/tx.go

```diff
@@ -83,7 +83,7 @@ type TxRequest[ADDR types.Hashable, TX_HASH types.Hashable] struct {
 
 	// Pipeline variables - if you aren't calling this from chain tx task within
 	// the pipeline, you don't need these variables
-	MinConfirmations  clnull.Uint32
+	MinConfirmations  clnull.Uint32 // deprecated
```

Comment:
Just marking as deprecate until other teams stop using the field. See [this eg. call](https://github.com/smartcontractkit/chainlink/blob/develop/core/services/pipeline/task.eth_tx.go#L160-L161)

Reply:
Is there a ticket or action item for another team to eventually remove this dependency? if there is none lets try to  make sure one exists and maybe link it as well in the deprecated comment

Reply:
Should not we respect the param if it was passed as part of task (job?) definition and only fallback to finality tags if was not specified?

Otherwise we are introducing changes that might affect UX (e.g. someone specified 1 as min number of confirmation, but we are waiting for 200 blocks)

Reply:
@poopoothegorilla Amit created one https://smartcontract-it.atlassian.net/browse/BCI-3962

Reply:
IIUC this format will be more widely recognized by tooling like linters, IDEs, etc.
```suggestion
	// Deprecated
	MinConfirmations  clnull.Uint32
```